syntax/javascript.vim

```diff
@@ -25,6 +25,7 @@ syntax sync fromstart
 syntax case match
 
 syntax match   jsNoise          /[:,\;\.]\{1}/
+syntax match   jsSemiEndOfLine  /\;\{1}\(\(\s*\/[\/\*].*\)\?$\)\@=/
```

Comment:
Just at a quick glance, I don't like this one at all because it's not contained AND has a look ahead. This will probably destroy performance in a larger file. Also, I am not quite sure the rationale to match end of line semis? Wouldn't it be better to isolate particular semis in contained groups we already have?